hawkular-bus-common/src/main/java/org/hawkular/bus/common/consumer/AbstractBasicMessageListener.java

```diff
@@ -125,7 +161,7 @@ public void run() {
                     });
                 }
             } else {
-                throw new Exception("Message is not a valid type: " + message.getClass());
+                throw new Exception("BasicMessage is not a valid type: " + message.getClass());
```

Comment:
I meant for this exception string to say the "message" parameter (of type javax.jms.Message) is not a valid type. This line change should be reverted, perhaps use all lowercase to make it clear that "'message' is not a valid type"